test/web.security/jwt.encryption.spec.unit.ts

```diff
@@ -0,0 +1,46 @@
+import * as jwt from "../../src/web.security/jwt.encryption";
+import {OAUTH2_REQUEST_KEY} from "../../src/properties";
+import * as jose from "node-jose";
+import {generateNonce} from "../../src/web.security/jwt.encryption";
+
+jest.mock("../../src/session/store/redis.store", () => {
```

Comment:
You might not have merged master in - you can use this mock syntax instead 
jest.mock("../../src/session/store/redis.store", () => import("../mocks/redis.store.mock.factory"));